docs/specification/compliance.md

```diff
@@ -2027,13 +2027,6 @@ cases where UCUM uses non-standard units (e.g. Cel, mo), as well as
 UCUM-specific syntax (e.g. `m.s-1`) and extensions (e.g. using
 annotations like `{Cre}`).
 
-*Developers note*: to run this test, the units extension should be
```

Comment:
This is unnecessarily confusing -- I just merged the `unit` dev deps into the main group so people don't have to dig through documentation to work out why their tests or their unit conversions aren't working.